setup.py

```diff
@@ -25,7 +25,9 @@ def read(*paths):
     'python-dateutil>=2.4.0',
     'rfc3986>=0.3.0',
     'jsonschema>=2.5.1',
-    'future>=0.15.2'
+    'future>=0.15.2',
+    'tabulator>=0.3',
+    'unicodecsv>=0.14',
```

Comment:
For now I suppose `yes` because we also need to write - https://github.com/datapackages/jsontableschema-py/blob/feature/plugins-and-storage/jsontableschema/resource.py#L92 (it's `unicodecsv` call)

I had some problems with py2-py3 writing and found a relief with this lib (or we need some additional code for it in the codebase).

I thins it's not a big deal to have it here for some time before https://github.com/datapackages/tabulator-py/issues/36 will be implemented. Then all csv interactions (read/write) could be encapsulated into `tabulator`.